src/app/sell/sell.modal.ts

```diff
@@ -0,0 +1,42 @@
+export type Sell = {
+  id: number
+  customerName: string
+  customerPhoneNumber: string
+  customerEmail: string
+  totalAmount: number
+  cake: {
+    name: string
+    descrition: string
+    ingredients: string[]
+    price: number
+    quantity: number
+  }
+}
+
+export interface SellInterface {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.

Since we are using a Single File Responsibility file distribution, this interface should live in a separate file named `cake-status.interface.ts`.